balena/models/environment_variables.py

```diff
@@ -209,10 +208,7 @@ def get_all(self, uuid):
 
         """
 
-        # TODO: pine client for python
-        device = self.device.get(uuid)
-
-        query = '$expand=service_install($select=id&$expand=service($select=service_name))&$filter=service_install/any(d:d/device%20eq%20{device_id})'.format(device_id=device['id'])
+        query = '$expand=service_install($select=id&$expand=service($select=service_name))&$filter=service_install/any(d:d/device/uuid%20eq%20{uuid})'.format(uuid=uuid)
```

Comment:
@Page- Any reason to prefer .format over (python f string)[https://peps.python.org/pep-0498/] like
```suggestion
        query = f'$expand=service_install($select=id&$expand=service($select=service_name))&$filter=service_install/any(d:d/device/uuid%20eq%20{uuid})'
```

Reply:
I was just modifying existing code (and have essentially non-existent python experience) but I'd be happy in general for the python SDK to use whatever is the best-practice/most idiomatic python features - I just have no idea what those would be